crates/electrum/Cargo.toml

```
@@ -9,12 +9,14 @@ description = "Fetch data from electrum in the form BDK accepts"
 license = "MIT OR Apache-2.0"
 readme = "README.md"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
 bdk_chain = { path = "../chain", version = "0.17.0" }
-electrum-client = { version = "0.20" }
-#rustls = { version = "=0.21.1", optional = true, features = ["dangerous_configuration"] }
+electrum-client = { version = "0.21", default-features = false }
 
 [dev-dependencies]
 bdk_testenv = { path = "../testenv", default-features = false }
+
+[features]
+default = ["use-rustls"]
+use-rustls = ["electrum-client/proxy", "electrum-client/use-rustls"]
+use-rustls-ring = ["electrum-client/proxy", "electrum-client/use-rustls-ring"]
```

Leaving a suggestion below, to make it easy for a quick change:

```suggestion
default = ["electrum-client/proxy", "use-rustls"]
use-rustls = ["electrum-client/use-rustls"]
use-rustls-ring = ["electrum-client/use-rustls-ring"]
```

---

Oh, scratch that ^

You can basically add it like `electrum-client = { version = "0.21", features = ["proxy"], default-features = false }` instead, right ?

---

Yep thanks for the suggestion. Committed.